### ISSUE TITLE
Input label opacity

### ISSUE THREAD

**Issue by kozakluke** [react-toolbox/react-toolbox#1343]
Hello, I want ask, why default opacity for Input label is only
0.26 it look not good, is too light 
here is preview from react-toolbox website
![image](https://cloud.githubusercontent.com/assets/3715762/24141846/063291ce-0e25-11e7-8cb9-44a732954f61.png)

and for compare from google material doc

![image](https://cloud.githubusercontent.com/assets/3715762/24141882/28ebec42-0e25-11e7-87a2-6ac208285211.png)

original is darker and looks better.

**Comment by drager** [comment]
Yeah, according to the [specification](https://material.io/guidelines/style/color.html#color-text-background-colors) it should be 38% #000000 on light backgrounds and 50% #FFFFFF on dark backgrounds.

**Comment by KhushilMistry** [comment]
Hey, I want to work on this. 
How can we find out that its light background or dark background ?
Or we should change only this for light backgrounds ?
```
--input-text-label-color: color(var(--color-text) a(26%));
```
to
```
--input-text-label-color: color(var(--color-text) a(38%));
```

**Comment by rubenmoya** [comment]
I think we have to review all colors in the project...

According to the Material Design Spec the bottom line should be `Color: 42% black` and the text should be `Color: 54% black` for normal inputs, and the text should be `Color: 38% black` for disabled inputs.

We're using `--color-text` which is `--palette-grey-900`, also known as `rgb(33, 33, 33)`, which is not black, so yeah, everything in the input looks lighter than it should...

**Comment by Rishi-VD** [comment]
Hey @kozakl, can you please assign this work for me
